crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,29 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+NAME:   dilly_ecumenical_altar_deadguardian
+TAGS:   no_monster_gen transparent
+MONS:   plant
+SUBST: z = t11
+KITEM: E = never_decay human skeleton / gnoll skeleton / elf skeleton / dwarf skeleton
+: dgn.delayed_decay(_G, "e","hoblin skeleton / hobgoblin skeleton / kobold skeleton")
+FTILE: 1et'_  = floor_grass
+: ecumenical_altar_setup(_G)
+MAP
+         t      
+        ttt     
+       tt_t     
+      tt'tt     
+     tt'ztt     
+     ttz'ttt    
```

Comment:
I feel like the path to the altar could be a bit shorter, since the player already passes through the tomb section on the way.

Reply:
Okay!